New External Storage page

jospoortvliet opened the pull request:
Not SUPER good, but a decent start I think:
![storage nextcloud 3-4-2017 21-06-06](https://cloud.githubusercontent.com/assets/551757/24626402/816983f2-18b1-11e7-90e4-26fb43ebf314.png)
@Espina2 @NinaNC would love to hear feedback on style and content.

NinaNC commented:
My two cents :)
![816983f2-18b1-11e7-90e4-26fb43ebf314](https://cloud.githubusercontent.com/assets/26001373/24649084/2c8efefa-1926-11e7-923c-04919fde6dc3.png)

Espina2 commented:
@jospoortvliet sounds good to me, just one tip on the Public, Private and hybrid unified the technologies you can put in one or two lines instead? 

Try to maintain the same space used for each item, and it will look better.

MariusBluem commented:
Could we also show some logos of the external storages instead of this list? 
This would reduce space-wasting on left and right side 😁 @jospoortvliet

MariusBluem commented on templates/header-top-navbar.php:
```diff
@@ -39,6 +39,7 @@
 						<li class="nav__item"><a href="<?php echo home_url('sharing') ?>"><?php echo $l->t('Sharing');?></a></li>
 						<li class="nav__item"><a href="<?php echo home_url('clients') ?>"><?php echo $l->t('Clients');?></a></li>
 						<li class="nav__item"><a href="<?php echo home_url('workflow') ?>"><?php echo $l->t('Workflow');?></a></li>
+						<li class="nav__item"><a href="<?php echo home_url('storage') ?>"><?php echo $l->t('Storage');?></a></li>
```
I think "External Storage" or something like "Storage integration" (for somebody who is not **that** technical) would describe this page better 😁

jospoortvliet replied:
I'd rather stick with one word as the others do... Already thinking about cutting 'Outlook integration' to 'Outlook'... But we should feature Outlook integration on nextcloud.com/features first so ppl find it easier.

jospoortvliet commented:
Here is the modified version, let me know what you think - it deals better with the issue @MariusBluem pointed out ;-)
![storage nextcloud 4-4-2017 18-42-28](https://cloud.githubusercontent.com/assets/551757/24668314/bd93e560-1966-11e7-8cff-e282e273727b.png)

MariusBluem commented:
Some feedback:
![bd93e560-1966-11e7-8cff-e282e273727b](https://cloud.githubusercontent.com/assets/19711361/24668635/ec34dad6-1967-11e7-86f6-8a576355a94b.png)

jospoortvliet commented:
Fixed headline and added a call to action... ;-)
![storage nextcloud 4-4-2017 19-40-35](https://cloud.githubusercontent.com/assets/551757/24670536/b95c499e-196e-11e7-8bdf-a039b4351a2c.png)

Espina2 commented:
It looks nice. Just one question, why you have that blue claim before the last point? 
@jospoortvliet
